config/schema/field_definitions.json

```diff
@@ -993,5 +993,20 @@
   "product_recall_alert_date":{
     "description": "Date of the recall/alert",
     "type": "date"
+  },
+  "category":{
```

Comment:
I wonder if it's useful to disambiguate these somewhat? I can see a future person coming along and assuming that category, publisher and date_published are generically used and are appropriate for co-opting into some other project? I think I'd have to do some digging as a new person to work out that date_published is specific to a particular document type.

Reply:
(And it's easier to do this now than once the thing is published!)

I suggest perhaps:
- digital_market_research_category
- digital_market_research_publisher
- digital_market_research_area
- digital_market_research_topic
- digital_market_research_publish_date (or see whether you can reuse another existing date field?)

Reply:
I've renamed all of these as suggested. As far as the date is concerned it doesn't look like we have a generic publication date field at all, various date fields exist but none seem directly equivalent. (Although maybe it might make sense to create one that future types could use.)